Update translation machinery

viccie30 opened the pull request:
`\notelabel` is used in the caption of figures and tables. According to the Dutch translation of APA that's endorsed by multiple Dutch higher education institutions, the word used should be 'Opmerking'. See for reference https://www.auteursrechten.nl/files/auteursrechten/2021-07/De%20APA-richtlijnen%20uitgelegd%20-%203e%20editie.pdf from page 87.

viccie30 committed:
Update Dutch translation

`\notelabel` is used in the caption of figures and tables. According to
the Dutch translation of APA that's endorsed by multiple Dutch higher
education institutions, the word used should be 'Opmerking'.
See for reference
https://www.auteursrechten.nl/files/auteursrechten/2021-07/De%20APA-richtlijnen%20uitgelegd%20-%203e%20editie.pdf
from page 87.

dan-weiss commented:
Thanks for this update! Please update lines 7 (I suggest 2.17), 10, and 11. Also add a change log entry around line 294.

viccie30 committed:
Use tracklang and translations packages to provide translations

viccie30 committed:
Update version number and changelog

viccie30 commented:
I have updated the branch and the title of this PR to also include a fix for #52. 

I have switched the translation machinery to use the tracklang and translation packages. Because translation does not allow choosing the filenames for its dictionary files, I have renamed the translation files and reset the version numbers back to 1.00.

It is necessary to include tracklang at the end of the preamble, because it does not pickup the languages passed directly to babel otherwise.

viccie30 commented:
I have now also updated the documentation to document that tracklang and translations packages are now used.
